functional/config.rb

```diff
@@ -0,0 +1,58 @@
+# Size of the CoreOS cluster created by Vagrant
```

Comment:
Do we need this configuration? It's just one more moving part. If we have already modified the Vagrantfile, then just do it in there and remove this file.